frontend/src/lib/components/common/confirmationModal/dirtyStore.ts

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store'
+
+export const dirtyStore = writable<boolean>(false)
```

Comment:
I think you do not gain much but having a common store for both flow and and script. Passing the store as a bind:props to the component should be good enough no ?

Reply:
Ah nevermind, you're doing it at the Editor level so it has to be global, gotcha